### PR TITLE
Feat/alias/config ccs line management

### PR DESCRIPTION
**Pull request by JeremyDec**
## CCS Number line management new pages
MANAGER-982

### Description of the Change
New pages for the contact center solution lines configuration

Two commits in this PR, first one is for deleted files that won't be used after
second one is the feature

### CODE REVIEW

File: client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/lines/telecom-telephony-alias-configuration-lines.controller.js [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+        value: strategy,
+      })),
+      status: _.get(models, 'telephony.OvhPabxHuntingAgentStatusEnum', { enum: [] }).enum.map(status => ({
+        label: this.$translate.instant(`telephony_alias_config_contactCenterSolution_lines_status_${status}`),
+        value: status,
+      })),
+    }));
+  }
+
+  upLineOrder(row) {
```
Comment (FredericEspiau) [inline client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/lines/telecom-telephony-alias-configuration-lines.controller.js]: Move to 1 method (`moveLineOrder`)

File: client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/lines/telecom-telephony-alias-configuration-lines.controller.js [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+    const deleteLineModal = this.$uibModal.open({
+      templateUrl: 'client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/lines/delete/telecom-telephony-alias-configuration-lines-delete.html',
+      controller: 'TelecomTelephonyAliasConfigurationLinesDeleteCtrl',
+      controllerAs: '$ctrl',
+      resolve: {
+        line,
+      },
+    });
+
+    deleteLineModal.result.then(() => this.deleteLine(line.agentId));
```
Comment (FredericEspiau) [inline client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/lines/telecom-telephony-alias-configuration-lines.controller.js]: Chaining

File: client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/lines/telecom-telephony-alias-configuration-lines.controller.js [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+    this.updatingAgents = true;
+    return this.tucVoipServiceAlias.deleteContactCenterSolutionNumberAgent(
+      this.serviceInfos,
+      agentId,
+    )
+      .then(() => this.tucVoipServiceAlias.fetchContactCenterSolutionNumberAgentsInQueue(
+        this.serviceInfos,
+        this.queue.queueId,
+      ))
+      .then((agents) => {
```
Comment (FredericEspiau) [inline client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/lines/telecom-telephony-alias-configuration-lines.controller.js]: Extract to method

File: client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/lines/telecom-telephony-alias-configuration-lines.controller.js [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+          this.contactCenterSolution,
+        )
+        : angular.noop(),
+      agentsList: this.canUpdateAgentsList() ? this.updateAgentsList() : angular.noop(),
+    }).then(() => {
+      this.OvhApiTelephony.EasyHunting().Hunting().v6().resetCache();
+      this.OvhApiTelephony.EasyHunting().Hunting().Queue().Agent()
+        .v6()
+        .resetAllCache();
+      this.$state.go('^').then(() => {
```
Comment (FredericEspiau) [inline client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/lines/telecom-telephony-alias-configuration-lines.controller.js]:
```suggestion
      return this.$state.go('^');
```

File: client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/lines/telecom-telephony-alias-configuration-lines.html [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+            <oui-action-menu data-align="end"
+                             data-compact>
+                <oui-action-menu-item data-text="{{:: 'modify' | translate }}"
+                                      data-on-click="$ctrl.updateLine($row)"></oui-action-menu-item>
+                <oui-action-menu-item data-text="{{:: 'delete' | translate }}"
+                                      data-on-click="$ctrl.deleteLineOpenModal($row)"></oui-action-menu-item>
+            </oui-action-menu>
+        </oui-datagrid>
+
+        <form name="options"
```
Comment (FredericEspiau) [inline client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/lines/telecom-telephony-alias-configuration-lines.html]: `$ctrl`

File: client/telecomUniverseComponents/telecom/voip/service/alias/voip-service-alias.service.js [hunk 6]
```diff
@@ -148,6 +148,245 @@ export default class {
...
+    }).then(agents => this.OvhApiTelephony.EasyHunting().Hunting().Queue().Agent()
+      .v6()
+      .getBatch({
+        billingAccount,
+        serviceName,
+        queueId,
+        agentId: _.pluck(agents, 'agentId'),
+      }).$promise.then((queues) => {
+        const orderedAgents = agents;
+        orderedAgents.map((agentParam) => {
```
Comment (FredericEspiau) [inline client/telecomUniverseComponents/telecom/voip/service/alias/voip-service-alias.service.js]:
```suggestion
        const orderedAgents  = agents.map((agentParam) => {
```

**Comment by JeremyDec** [comment]
@cbourgois As we seen together, I will transfer voip-service-alias.service.js into old component directory (instead of migrating it into telecomUniverseComponents). I'll do it once this PR will be merged, to avoir time loss because of conflicts

**Pull request merged**